debian/vzlogger.init

```diff
@@ -14,7 +14,7 @@
 PATH=/sbin:/usr/sbin:/bin:/usr/bin
 DESC="smartmetering server"
 NAME=vzlogger
-DAEMON=/usr/local/bin/vzlogger
+DAEMON=/usr/bin/vzlogger
```

Comment:
thanks for removing this abomination.
does the rest of the setup also take care to install the binary in the expected location?

Reply:
This is only for the debian package. I have not touched the remaining install process, e.g. everything in the non-debian directories is still as-is. I know this works for debian, because it uses the right prefix.